lib/InputFilesGenerator.ex

```diff
@@ -12,38 +13,55 @@ defmodule InputFilesGenerator do
   ```
   where:
   + sequence_id - the ordering number of the packet (relative to each of the peers). Basing on this number we will be able to assemble the particular frame.
-  + frame_id - the identifier which consists of the number of the frame to which the body of a given packet belongs, optionally followed by a single character **'s'** (meaning that this packet is a **s**tarting packet of a frame) or by **'e'** character (meaning that the packet is the **e**nding packet of the frame). Note that frames are numbered relatively to each peer in that conversation and that frame_id does not describe the global order of the frames in the final file.
+  + frame_id - the identifier which consists of the number of the frame to which the body of a given packet belongs,
+  optionally followed by a single character **'e'** (meaning that the packet is the **e**nding packet of the frame).
```

Comment:
Is it intentional to remove that part of a sentence: "...single character **'s'** (meaning that this packet is a **s**tarting packet of a frame)..."?